packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -566,18 +638,60 @@ export class MgtTodo extends MgtTasksBase {
     }
   }
 
+  private handleTaskCheckKeydown(e: KeyboardEvent, task: TodoTask) {
+    if (e.key === 'Enter' && !this.readOnly) {
+      this.handleTaskClick(task);
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
We shouldn't need this suppression, or the one above it in `handleTaskCheckClick`